workspaces/liferay-learn-workspace/client-extensions/liferay-learn-site-initializer/site-initializer/fragments/group/liferay-learn/fragments/article-related-recipes/index.js

```diff
@@ -0,0 +1,82 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+async function createRecipeSuggestions() {
+	const articleId = document.querySelector(
+		'.article-related-recipes'
+	).dataset.articleId;
+
+	const structuredContent = await Liferay.Util.fetch(
+		`/o/headless-delivery/v1.0/sites/${Liferay.ThemeDisplay.getSiteGroupId()}/structured-contents/by-key/${articleId}`
+	).then((response) => response.json());
+
+	if (structuredContent.keywords.length) {
+		createRecipesContainer();
+		await getRecipeKeywordsSearch(structuredContent.keywords);
+	}
+}
+
+async function getRecipeKeywordsSearch(articleKeywords) {
```

Comment:
this naming doesn't seem accurate
The current function is doing more than just a get
let's try to rename it